src/org/pentaho/di/trans/steps/cassandrasstableoutput/SSTableOutputMeta.java

```diff
@@ -229,6 +250,9 @@ public String getXML() {
           XMLHandler.addTagValue("buffer_size_mb", bufferSize));
     }
 
+    retval.append("\n    ").append( //$NON-NLS-1$
```

Comment:
This method should be tested via a unit test. And this change should cause an update to that ttest (but I don't see it). If it does, update it so that the tests pass. If not, please create one.